cmd/aries-agentd/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/hyperledger/aries-framework-go/cmd/aries-agentd
```

Comment:
You need to remove the dependency from go.mod root

Reply:
done